awx/templates/rest_framework/api.html

```diff
@@ -1,11 +1,18 @@
 {% extends 'rest_framework/base.html' %}
-{% load i18n static %}
+{% load i18n static ansible_base.lib.templatetags.requests ansible_base.lib.templatetags.util %}
 
 {% block title %}{{ name }} &middot; {% trans 'AWX REST API' %}{% endblock %}
 
 {% block bootstrap_theme %}
+  {% is_proxied_request as proxied %}
   <link rel="stylesheet" type="text/css" href="{% static 'rest_framework/css/bootstrap.min.css' %}" />
+  {% if proxied %}
+  <style>
+    {% inline_file "static/api/api.css" True %}
```

Comment:
Let's just leave a little teeny tiny comment saying that `inline_file` is a DAB util. You know, just for purposes of back-linking, so I won't go search Django source code for it.